### ISSUE TITLE
Volume bar not shown on android devices

### ISSUE THREAD

**Issue by gabrieletassoni** [openplayerjs/openplayerjs#324]
**Describe the bug**
AS per attachement: during lives on android smartphones, the volume control is not present. This way, if the video starts muted, there's no way, during live broadcasts to raise it.
![Screenshot_20220206-190542](https://user-images.githubusercontent.com/11076471/152694982-d9a05e96-256f-4119-895c-d04986c7e51d.png)

**Comment by gabrieletassoni** [comment]
Is not only during lives, also normal streaming the volume does not appear.
On PCs the volume is shown.
--
![Screenshot_20220206-194559](https://user-images.githubusercontent.com/11076471/152696763-0bbfd23d-8896-4262-b52a-082eafe88ed6.png)

**Comment by rafa8626** [comment]
@gabrieletassoni the volume control is never shown on mobile devices since it favors the native phone controls, in both Android and iOS. This is something various players follow for small devices

**Comment by gabrieletassoni** [comment]
Ah, ok, understood, is there a way to force it be shown? A customer asks for it to be visible anyway. Maybe a css hack, just need to be pointed in the right direction. Otherwise, thanks anyway, always a great player 🙂

**Comment by gabrieletassoni** [comment]
There's just something I'm missing in the usage, not having volume control on mobile devices, misses the part when a video is set as autoplay, in that case the video must be muted, otherwise autoplay won't work if no interaction occurred on the page. In that case, there's no way to at least unmute it and the user experience would be of just video and no audio, am I missing something?

Thank again

**Comment by rafa8626** [comment]
It should have displayed a Tap here to unmute button, but if it’s needed I can create a configuration element to show it on mobile

**Comment by gabrieletassoni** [comment]
Unfortunately there's no "Tap Here" button, if you need to test: https://www.alchemic.it/prova.html there is a streaming iframe which presents the problem on Android devices

**Comment by rafa8626** [comment]
I forgot to mention that the Tap here button only appears when you attempt to autoplay the video; otherwise, you won’t see it.

**Comment by gabrieletassoni** [comment]
> I forgot to mention that the Tap here button only appears when you attempt to autoplay the video; otherwise, you won’t see it.

Understood, so in my case (this specific installation didn't wanted the autoplay at all), the muted video would have remained muted anyway, maybe a configuration override which always shows the unmute, in case of muted audio, would have been a lifesaver in this case.

**Comment by rafa8626** [comment]
@gabrieletassoni is there a way you can test this in the `v3.0.0` branch and let me know if it works? I've been trying to complete most of the unit tests there, but if I have a user that starts using it, I can launch the version 3 and continue working on more unit tests from that point forward. If so, please let me know so I can schedule having this and the other enhancement ready for you in that branch

**Comment by rafa8626** [comment]
@gabrieletassoni To do this use the new useNativeVolume option as false in the v3.0.0 branch; let me know if it works for you

**Comment by gabrieletassoni** [comment]
Yes, I can test it in the next hours, I'll let you know, thank you very much for the effort

**Comment by gabrieletassoni** [comment]
Hi, sorry for late reply, today I managed to test.
this is the definition I use for the player:

```javascript
const player = new OpenPlayerJS('player', {
    showLoaderOnInit: false,
    pauseOthers: false,
    detachMenus: true,
    mode: 'fit',
    startVolume: 0, 
    useNativeVolume: false,
    hls: {
        limitRenditionByPlayerDimensions: false,
        smoothQualityChange: true,
        overrideNative: true
    },
});
```

But I still cannot see a way to control the volume (or just the mute unmute) on a android device:

![Screenshot_20220211-160447](https://user-images.githubusercontent.com/11076471/153616217-fb1456e2-43d5-4523-814a-9c55e1239124.png)

**Comment by rafa8626** [comment]
@gabrieletassoni I may have forgotten to create the bundle for this; I’ll check that and get back to yoh

**Comment by rafa8626** [comment]
@gabrieletassoni my bad, I changed the name to something more intuitive: `useDeviceVolume`. Please test using that property

**Comment by gabrieletassoni** [comment]
Tnx, on Monday I'll check it, now I don't have my PC with me.

**Comment by rafa8626** [comment]
Closing this and to be released in 3.0 version
